src/pages/insights/[post]/index.tsx

```diff
@@ -26,42 +26,44 @@ export const getStaticPaths = async () => {
 
 export const getStaticProps = async (context: { params: { post: string } }) => {
   const path = context.params?.post;
-  return { props: { path } };
+  const data = await fetch(`https://defi.sucks/blog-posts/${path}.md`);
+  const dataText = await data.text();
+  return { props: { path, blog: dataText } };
 };
 
-export default function Posts({ path }: InferGetStaticPropsType<typeof getStaticProps>) {
-  const id = path;
+export default function Posts({ path, blog }: InferGetStaticPropsType<typeof getStaticProps>) {
   const postData = posts.filter((post) => post.id === path)[0];
 
-  const [blog, setBlog] = useState('');
+  const [blogText, setBlogText] = useState('');
+
+  const post = blog || blogText;
 
   useEffect(() => {
-    fetch(`/blog-posts/${id}.md`)
-      .then((response) => response.text())
-      .then((data) => {
-        setBlog(data);
-      });
-  }, [id]);
+    if (!blog)
```

Comment:
Use {} for standard

Reply:
fixed https://github.com/defi-wonderland/web/pull/84/commits/f1db9577b1b61fadb8c417322a02c4e811c60ee5